enterprise/setup/index.md

```diff
@@ -0,0 +1,323 @@
+---
+title: 'Enterprise Edition: Initial Setup'
+backlink_href: /docs/enterprise
+backlink_title: 'pganalyze Enterprise Edition'
+---
+
+These are the installation instructions for [pganalyze Enterprise Edition](/enterprise-postgres-monitoring), which runs fully-contained within your own environment.
+
+The pganalyze Docker image is mostly self-contained, and runs various services inside the Docker container.
+
+```toc
+```
+
+## Installation steps
+
+### Pre-requisites
+
+* Provision a virtual machine that has at least 4GB - 8GB of RAM, 1-2 dedicated CPU cores
+* Install Docker on your virtual machine (pganalyze Enterprise Edition is delivered through a Docker image)
+* PostgreSQL database that can be used for storing statistics data (50 GB of disk space or more)
```

Comment:
We should specify minimum Postgres version here. Also, the other bullet points are "\<take this action\>" and this is "\<software that needs to be installed\>"; we should make this consistent.